### ISSUE TITLE
asRawJson() option in ResourceRepresentation

### ISSUE THREAD

**Issue by holtonma** [jesperfj/force-rest-api#47]
https://github.com/jesperfj/force-rest-api/blob/0c224e0f90a1dc4c3a751a4f10a4c6899ad6f09d/src/main/java/com/force/api/ResourceRepresentation.java#L29

In addition to `asList()` and `asMap()` and the custom `as(Class<T>)` it would be nice to expose `asRawJson()` method or similar.  This way one could receive the raw JSON from the endpoint, and deserialize in other/custom ways.  Perhaps there's a clean way to do this which I am not seeing, if so any insight in that direction is appreciated.  Thank you for the work on this library.

**Comment by jesperfj** [comment]
Yeah. At first glance that should be straightforward and fits well with the
architecture where ResourceRepresentation delays deserialization

**Comment by jesperfj** [comment]
Haven't had a chance to look at this in a while. Pull requests are much welcome! :-)
